### DISCUSSION TITLE
Config options for the clue helper plugin

### DISCUSSION THREAD

**Discussion by puhtahtoe** [runelite/runelite#15589] [Ideas]
I love most of the clue helper but I like to solve puzzle boxes myself. It'd be cool if the clue helper had the option to enable/disable helpers for different types of clue. Bonus if showing a reference picture for the clue box is another option.

**Comment by NotBlackice** [comment]
I noticed there's already an existing pull request for this feature from 2018, but it needed a typo fixed and never got merged. https://github.com/runelite/runelite/pull/5773

Hopefully someone gets around to it.
